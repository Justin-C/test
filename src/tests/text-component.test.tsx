import React from 'react';

import Text from '../components/text';
import renderer from 'react-test-renderer';

describe('text component test', () => {
  it('should render text', () => {
    const tree = renderer.create(<Text textString="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
