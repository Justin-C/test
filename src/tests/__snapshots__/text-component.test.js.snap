// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text component test should render text 1`] = `
<p
  className="undefined text--default"
>
  test
</p>
`;
